chemprop/args.py

```diff
@@ -229,6 +233,10 @@ class TrainArgs(CommonArgs):
     """
     ignore_columns: List[str] = None
     """Name of the columns to ignore when :code:`target_columns` is not provided."""
+    atom_targets: List[str] = []
+    """Name of the columns containing target atomic values."""
+    bond_targets: List[str] = []
+    """Name of the columns containing target bond values."""
```

Comment:
Would these be better suited as TrainArgs? With a separate argument in PredictArgs?